sources/gdal/large_image_source_gdal/__init__.py

```diff
@@ -358,6 +358,15 @@ def getHexColors(palette):
         try:
             return attrgetter(palette)(palettable).hex_colors
         except AttributeError:
+            pass
+        try:
+            # Attempt getting palette by MPL cmap name, e.g. 'viridis' or 'jet'
+            import matplotlib
+            import matplotlib.colors as mcolors
+
+            cmap = matplotlib.cm.get_cmap(palette, 255)  # 255 might be overkill
```

Comment:
This is probably major overkill. @manthey, any idea what might be a more reasonable value here?

Reply:
You don't need to specify the length.  That is,
`cmap = matplotlib.cm.get_cmap(palette)` will yield the same as `cmap = matplotlib.cm.get_cmap(palette, len(maplotlib.cm.get_cmap(palette).colors))`